docs/administer/rbac.md

```diff
@@ -0,0 +1,552 @@
+# Role Based Access Control
```

Comment:
Let's add a banner saying that this feature is in tech-preview stage

Reply:
Fixed.